### PR TITLE
feat: update gcc to 14.2, bump NVIDIA driver versions

### PR DESCRIPTION
**Pull request by smira**
Bump NVIDIA driver versions to match the `pkgs`.

### CODE REVIEW

File: nvidia-gpu/nvidia-fabricmanager/production/manifest.yaml
Comment (frezbo) [inline nvidia-gpu/nvidia-fabricmanager/production/manifest.yaml:1]: weird

**Review by frezbo — APPROVED** [review]

File: nvidia-gpu/nvidia-fabricmanager/production/manifest.yaml
Comment (smira) [reply nvidia-gpu/nvidia-fabricmanager/production/manifest.yaml:1]: I just moved things, you had a path like `lts/production` ;)

Comment (frezbo) [reply nvidia-gpu/nvidia-fabricmanager/production/manifest.yaml:1]: ahhh :facepalm: , thanks for fixing it

**Pull request merged**